spec/services/feeds/import_spec.rb

```diff
@@ -63,28 +63,54 @@
       end
     end
 
-    # it "reports an article creation error" do
-    #   allow(described_classs).to receive(:create_articles_from_user_feed).and_raise(StandardError)
-    #   allow(Honeybadger).to receive(:notify)
+    it "queues as many slack messages as there are articles", vcr: { cassette_name: "feeds_import" } do
+      old_count = Slack::Messengers::Worker.jobs.count
+      num_articles = described_class.call
+      expect(Slack::Messengers::Worker.jobs.count).to eq(old_count + num_articles)
+    end
 
-    #   described_class.call
+    context "when handling errors", vcr: { cassette_name: "feeds_import" } do
+      it "reports an article creation error" do
+        allow(Article).to receive(:create!).and_raise(StandardError)
+        allow(Rails.logger).to receive(:error)
 
-    #   expect(Honeybadger).to have_received(:notify).at_least(:once)
-    # end
+        described_class.call
 
-    # it "reports a fetching error" do
-    #   allow(rss_reader).to receive(:fetch_feeds).and_raise(StandardError)
-    #   allow(Honeybadger).to receive(:notify)
+        expect(Rails.logger).to have_received(:error).at_least(:once)
+      end
 
-    #   described_class.call
+      it "reports a fetching error" do
+        allow(HTTParty).to receive(:get).and_raise(StandardError)
+        allow(Rails.logger).to receive(:error)
 
-    #   expect(Honeybadger).to have_received(:notify).at_least(:once)
-    # end
+        described_class.call
 
-    it "queues as many slack messages as there are articles", vcr: { cassette_name: "feeds_import" } do
-      old_count = Slack::Messengers::Worker.jobs.count
-      num_articles = described_class.call
-      expect(Slack::Messengers::Worker.jobs.count).to eq(old_count + num_articles)
+        expect(Rails.logger).to have_received(:error).at_least(:once)
+      end
+
+      it "reports a parsing error" do
+        allow(Feedjira).to receive(:parse).and_raise(StandardError)
+        allow(Rails.logger).to receive(:error)
+
+        described_class.call
+
+        expect(Rails.logger).to have_received(:error).at_least(:once)
+      end
+    end
+
+    context "with an explicit set of users", vcr: { cassette_name: "feeds_import" } do
+      it "accepts a subset of users" do
+        num_articles = described_class.call(users: User.with_feed.limit(1))
+
+        verify(format: :txt) { num_articles }
```

Comment:
Totally random question here, why do you choose to use a `verify` here rather than `expect(num_articles).to eq(10)` ?